mayastor/src/bdev/dev/nvmx/channel.rs

```diff
@@ -63,24 +65,75 @@ pub struct NvmeIoChannelInner<'a> {
     pub qpair: *mut spdk_nvme_qpair,
     poll_group: NonNull<spdk_nvme_poll_group>,
     poller: poller::Poller<'a>,
+    // Flag to indicate the shutdown state of the channel.
```

Comment:
I suggest using `///` based doc comments here so IDE tooling! (It's not public, so it wouldn't show up in generated docs, but IDE tools can helpful and it's nearly for free)

Reply:
Indeed it's useful for IDE/tools. Thanks !